### PR TITLE
riemann-client 2.1.0

### PR DESCRIPTION
**Pull request by chenrui333**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

deprecate the formula as repo got removed.

### CODE REVIEW

**Review by iMichka — CHANGES_REQUESTED** [review]
`collectd` still depends on `riemann-client`. We need to find a solution for it first before deprecating.

**Comment by chenrui333** [comment]
> `collectd` still depends on `riemann-client`. We need to find a solution for it first before deprecating.

I think we should deprecate `collectd` as well.

**Comment by cho-m** [comment]
Ref: https://github.com/algernon/

> Repositories are being migrated to my own [Gitea instance](https://git.madhouse-project.org/algernon), you'll find them there. Below is a non-exhaustive list of my things that used to be hosted on GitHub, but no longer are:
> 
> - [riemann-c-client](https://git.madhouse-project.org/algernon/riemann-c-client)

File: Formula/riemann-client.rb [hunk 1]
```diff
@@ -1,10 +1,10 @@
 class RiemannClient < Formula
   desc "C client library for the Riemann monitoring system"
   homepage "https://github.com/algernon/riemann-c-client"
```
Comment (cho-m) [inline Formula/riemann-client.rb:3]:
```suggestion
  homepage "https://git.madhouse-project.org/algernon/riemann-c-client"
```

File: Formula/riemann-client.rb [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
-  url "https://github.com/algernon/riemann-c-client/archive/riemann-c-client-2.0.1.tar.gz"
-  sha256 "2ed963eeef5517f7be921790ef0cde13c7bcac172ac14ce5818d84a261cc3b31"
+  url "https://git.madhouse-project.org/algernon/riemann-c-client/archive/riemann-c-client-2.1.0.tar.gz"
+  sha256 "e1a4439ee23f4557d7563a88c67044d50c384641cf160d95114480404c547085"
   license "LGPL-3.0-or-later"
-  head "https://github.com/algernon/riemann-c-client.git", branch: "master"
+  head "https://git.madhouse-project.org/algernon/riemann-c-client.git", branch: "master"
```
Comment (cho-m) [inline Formula/riemann-client.rb:7]:
```suggestion
  head "https://git.madhouse-project.org/algernon/riemann-c-client.git", branch: "main"
```

**Review by SMillerDev — APPROVED** [review]

**Review by iMichka — APPROVED** [review]

**Pull request closed without merging**